core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.expression.operator.arthmetic;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class MathUtil {
+
+  /**
+   * Truncates a number to required decimal places.
+   *
+   * @param numberToTruncate number to be truncated
+   * @param numberOfDecimals required decimal places
+   * @return truncated number as {@link BigDecimal}
+   */
+  public static BigDecimal truncateNumber(double numberToTruncate, int numberOfDecimals) {
+    if (numberToTruncate > 0) {
```

Comment:
Better to use a conditional instead of if...else:
```
return new BigDecimal(String.valueOf(numberToTruncate)).setScale(numberOfDecimals,
            numberToTruncate > 1.0 ? RoundingMode.FLOOR : RoundingMode.CEILING);
```
or 

```
RoundingMode roundingMode = numberToTruncate > 1.0 ? RoundingMode.FLOOR : RoundingMode.CEILING;
    BigDecimal bd =  new BigDecimal(String.valueOf(numberToTruncate)).setScale(numberOfDecimals, roundingMode);
```